docs/user/managing-devices.md

```diff
@@ -644,15 +681,257 @@ spec:
+            [Volume]
+            VolumeName=postgres-data
+
+        - path: postgres.container
+          content: |
+            [Container]
+            Image=quay.io/library/postgres:16
+            Volume=db.volume:/var/lib/postgresql/data:Z
+            Environment=POSTGRES_PASSWORD=secret
+
+            [Service]
+            Restart=always
+
+            [Install]
+            WantedBy=default.target
+```
+
+**Multi-Container with Network:**
+
+```yaml
+spec:
+  applications:
+    - name: web-app
+      appType: quadlet
+      inline:
+        - path: app-net.network
+          content: |
+            [Network]
+            NetworkName=app-network
+
+        - path: backend.container
+          content: |
+            [Container]
+            Image=quay.io/myorg/backend:v1
+            Network=app-net.network
+
+        - path: frontend.container
+          content: |
+            [Container]
+            Image=quay.io/myorg/frontend:v1
+            Network=app-net.network
+            PublishPort=8080:80
+```
+
+**Environment Variables:**
+
+Environment variables are injected using the `envVars` field. The agent generates a `.env` file and systemd drop-in configuration to apply them:
+
+```yaml
+spec:
+  applications:
+    - name: api-server
+      appType: quadlet
+      envVars:
+        DATABASE_HOST: "db.internal"
+        API_PORT: "8080"
+        LOG_LEVEL: "info"
+      inline:
+        - path: api-server.container
+          content: |
+            [Container]
+            Image=quay.io/myorg/api-server:v2.1
+            PublishPort=8080:8080
+```
+
+**Volume from OCI Image:**
+
+Volumes can be populated from OCI images, useful for delivering configuration files or data:
+
+```yaml
+spec:
+  applications:
+    - name: config-app
+      appType: quadlet
+      inline:
+        - path: config.volume
+          content: |
+            [Volume]
+            Image=quay.io/myorg/app-config:v1
+
+        - path: app.container
+          content: |
+            [Container]
+            Image=quay.io/myorg/myapp:v1
+            Volume=config.volume:/config:ro
+```
+
 ### Adding Application Volumes
 
 > [!NOTE]
 > This feature requires the Flight Control Agent to run with **Podman version 5.5 or higher**.
 
-Applications can declare persistent data volumes that are populated from OCI artifacts. This allows delivering large datasets (such as ML models or static assets) as part of the application deployment.
+Compose applications can declare persistent data volumes that are populated from OCI artifacts. This allows delivering large datasets (such as ML models or static assets) as part of the application deployment.
 
 Volumes are declared under each application in the `volumes` field. These volumes are mounted into the application containers via the Compose `volumes` section.
 
+> [!NOTE]
+> Quadlet applications use native volume definitions (`.volume` files) instead of the `volumes` field. See the Quadlet examples above for volume usage.
```

Comment:
:+1:

Reply:
its sort of awkward the quadlet does not support artifact. could we use the volume applications API to populate a path local to the application and then translate that into a quadlet volume behind the scene?

Reply:
Just wait for podman 6. 

I'll look into something.